source/_integrations/zha.markdown

```diff
@@ -132,12 +132,15 @@ Some other Zigbee coordinator hardware may not support a firmware that is capabl
   - [ZiGate-Ethernet (Ethernet gateway board for PiZiGate)](https://zigate.fr/produit/zigate-ethernet/)
   - [ZiGate + WiFi Pack](https://zigate.fr/produit/zigatev2-pack-wifi/)
 
-#### Warning about Wi-Fi-based Zigbee-to-Serial bridges/gateways
+#### Warning about using Zigbee Coordinator over WiFi/WAN/VPN
 
 <div class="note warning">
```

Comment:
Not changed that line.

Reply:
Someone else needs to make that determination, and regardless, that change should be done in a separate pull request

I did not change any formatting as do not want this pull request to be held up based on waiting for decisions on that.

Reply:
> Would you like me to open a new GitHub issue to track the suggestion of replacing HTML with Markdown for future discussion and determination? This way, it can be addressed independently without impacting the current pull request's progress. Let me know how you'd like to proceed!

Yes you can open a new GitHub issue to track the suggestion of replacing HTML with Markdown for future discussion and determination